src/components/play/layer-settings/LayerListPopup.js

```diff
@@ -19,7 +19,7 @@ const LayerListPopup = ({
     user,
     round
 }) => (
-    <Box className={showMixerPopup ? classes.mixerPopup : classes.hidden}>
+    <Box className={showMixerPopup ? classes.mixerPopup : classes.hidden} style={height < 400 ? { height: height - 142, top: -(height - 138) } : {}}>
```

Comment:
oof, this looks a bit brittle, but if it works it works I guess

Reply:
Could be done in a better way I guess, I'll clean it up a little.